src/pages/sections/CurrentlyReading/CurrentlyReading.tsx

```diff
@@ -0,0 +1,6 @@
+import './CurrentlyReading.scss';
+export const CurrentlyReading = () => (
+  <div className="currently-reading-section">
+    <h2 className="heading2">what are we currently reading?</h2>
+  </div>
```

Comment:
The styles with h2 are repeated like 6 times. Consider abstracting it to a component.